holmes_operator/operator.py

```
@@ -0,0 +1,112 @@
+import sys
+
+import kopf
+from kubernetes import client
+from kubernetes import config as k8s_config
+
+from holmes_operator import context
+from holmes_operator.client.holmes_api_client import HolmesAPIClient
+from holmes_operator.config import OperatorConfig
+
+# Import handlers to register them with kopf
+from holmes_operator.handlers import healthcheck  # noqa: F401
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)